Formula/m/manticoresearch.rb

```diff
@@ -3,7 +3,7 @@ class Manticoresearch < Formula
   homepage "https://www.manticoresearch.com"
   url "https://github.com/manticoresoftware/manticoresearch/archive/refs/tags/6.2.12.tar.gz"
   sha256 "272d9e3cc162b1fe08e98057c9cf6c2f90df0c3819037e0dafa200e5ff71cef9"
-  license "GPL-2.0-only"
+  license "GPL-2.0-only" # License changes in the next release and must be removed from formula_license_mismatches
```

Comment:
This is definitely an audit bug if it's considering `master` and not the tag.

Reply:
Agreed, could one of you file a Homebrew/brew issue @SMillerDev @Bo98?